WalletWasabi/CoinJoin/Coordinator/Rounds/CoordinatorRound.cs

```diff
@@ -1199,11 +1214,16 @@ public async Task<IEnumerable<Alice>> RemoveAlicesIfAnInputRefusedByMempoolAsync
 						Logger.LogInfo($"Mempool acceptance failed: {result.reason ?? "no reason"}.");
 					}
 				}
-			}
 
-			foreach (var alice in alicesRemoved)
-			{
-				Logger.LogInfo($"Round ({RoundId}): Alice ({alice.UniqueId}) removed.");
+				foreach (var alice in alicesRemoved)
+				{
+					Logger.LogInfo($"Round ({RoundId}): Alice ({alice.UniqueId}) removed.");
+				}
```

Comment:
If the above comment is accepted, we can remove this foreach.

Reply:
I am not sure. Probably better to log in this way for some reason. Changing this would change the order of the log entries.